containers/ddev-dbserver/README.md

```diff
@@ -1,10 +1,64 @@
-# ddev-dbserver for ddev
+# ddev-dbserver docker image
 
-This Docker image builds database server images (MariaDB and MySQL) for DDEV.
+## Overview
 
+Docker container image for DDEV's ddev-dbserver container. This same code is used to build all of the MariaDB and MySQL images.
+
+This container image is part of DDEV, and not typically used stand-alone.
+
+### Features
+
+* MariaDB 5.5 though current stable
+* MySQL 5.5 through current stable
+* Backup facilities like xtrabackup and mariabackup.
+
+## Instructions
+
+Use [DDEV](https://ddev.readthedocs.io)
+
+### Building and pushing to Docker Hub
+
+See [DDEV docs](https://ddev.readthedocs.io/en/stable/developers/release-management/#pushing-docker-images-with-the-github-actions-workflow)
+
+### Running
+To run the container by itself:
+
+```bash
+docker run -it --rm --entrypoint=bash ddev/ddev-webserver:v1.22.7 bash
```

Comment:
```suggestion
docker run -it --rm --entrypoint=bash ddev/ddev-db-server-<mariadb|mysql>-<version>:<tag>
```

I removed `bash` from the end because there was an error in `docker run`:

```
/usr/bin/bash: /usr/bin/bash: cannot execute binary file
```